src/main/java/net/dv8tion/jda/bot/sharding/DefaultShardManager.java

```diff
@@ -647,9 +649,9 @@ protected JDAImpl buildInstance(final int shardId) throws LoginException, Interr
 
         jda.asBot().setShardManager(this);
 
-        if (this.eventManager != null)
-            jda.setEventManager(this.eventManager);
-
+        if(this.eventManagerProvider != null)
```

Comment:
Can you add the space back that you removed here between `if` and the parenthesis

Reply:
Also tab -> space in the line below

Reply:
fuck eclipse